cli/tests/unit_cli.yaml

```diff
@@ -20,9 +20,8 @@ services:
   cli:
     image: sawtooth-dev-test:$ISOLATION_ID
     volumes:
-      - ../../:/project/sawtooth-core
-    working_dir: /project/sawtooth-core/
-    entrypoint: nose2-3 -s /project/sawtooth-core/cli/tests
+      - $SAWTOOTH_CORE:/project/sawtooth-core
+    command: nose2-3 -v -s /project/sawtooth-core/cli/tests
```

Comment:
Is it necessary to have complete path?  Unless this is here to prevent ambiguity if there were `tests` directory in more than one of the directories in `PYTHONPATH`.